.github/workflows/ci.yml

```diff
@@ -167,6 +167,15 @@ jobs:
         key: ${{ runner.os }}-test-old-versions-1.12.0
         restore-keys: ${{ runner.os }}-test-old-versions-1.11.0
 
+    - name: Install Toolkit
+      run: |
+        (
+          cd extension
+          cargo pgx install -c /usr/lib/postgresql/12/bin/pg_config
```

Comment:
That's what https://github.com/timescale/timescaledb-toolkit/commit/1ffa4e9d02ed9dd1011f4200224ab4d4e535a8c8#diff-018192a8c8e7dea0df3dbbfdc766bfad7a3017167956b9714f8653b5f7644f72R167 is supposed to do - is it not working?

I did admit that was only lightly tested :)

We should fix it rather than trying to work around it, though.

Reply:
I didn't realize that `testbin` was supposed build Toolkit. I added that after seeing [this failed run](https://github.com/timescale/timescaledb-toolkit/actions/runs/3192484059/jobs/5210012286): it had an error since it couldn't find a control file for Toolkit. I think the problem is that `test_ci` calls `deb_start_test` which calls `start_test` which creates test objects before Toolkit is installed.

Reply:
No, you made the line that installs the old version not run in 'ci' mode; that's why you found no toolkit installed in `start_test`.